samples/PackageMapSample/UpgradeStepSample/ExtensionManifest.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  // This optional property gives a friendly name used for the extension
+  // while running and logging output from Upgrade Assistant.
+  "ExtensionName": "Custom package map extension sample",
+
+  // The optional PackageUpdater section of an extension manifest lists settings
+  // relevant to the packge upgrade step.
+  "PackageUpdater": {
+    // PackageMapPath indicates the folder (relative to this config file) where
+    // package maps should be loaded from.
+    "PackageMapPath": "SamplePackageMaps"
+  }
+
+  // Because this extension doesn't have any binary components, it doesn't
+  // need to be compiled. If your extension does have binaries that need
```

Comment:
wondering about the words `that need built`. Do we actually build the extender's code?

Reply:
In other cases, the extender needs to build their own code and include the binaries in their extension. In the case of updating package maps, that step isn't necessary since there is no code.